doc/isomorph.xml

```diff
@@ -96,20 +199,19 @@ false]]></Example>
       to the semigroup <A>T</A>. If it finds one, then it is returned, and if
       not, then <K>fail</K> is returned. <P/>
 
-      For many types of semigroup, <C>IsomorphismSemigroups</C> is not able
-      to determine whether or not <A>S</A> and <A>T</A> are isomorphic, and so
-      this operation may result in an "Error, no method found".
-      <C>IsomorphismSemigroups</C> may be able deduce that <A>S</A> and <A>T</A>
-      are not isomorphic by finding that some of their semigroup-theoretic
-      properties differ; however it is harder to construct an isomorphism for
-      semigroups that are isomorphic. <P/>
-
-      At present, <C>IsomorphismSemigroups</C> is only able to return an
-      isomorphism when <A>S</A> and <A>T</A> are finite simple, 0-simple, or
-      monogenic semigroups, or when <C><A>S</A> = <A>T</A></C>. 
-      See <Ref Prop="IsSimpleSemigroup" />, <Ref Prop="IsZeroSimpleSemigroup"
-        />, and <Ref Prop="IsMonogenicSemigroup" /> for more information
-      about these types of semigroups.
+      <C>IsomorphismSemigroups</C> uses the machinery for finding isomorphisms
+        between vertex coloured digraphs in &bliss; via 
+      <Ref Oper="IsomorphismDigraphs" 
+           Label="for digraphs and homogeneous lists"
+           BookName="digraphs"/> using digraphs constructed from the
+        multiplication tables of <A>S</A> and <A>T</A>.
+      <P/>
+
+      Note that finding an isomorphism between two semigroups is difficult, and
+      may not be possible for semigroups whose size exceeds approximately
+      2048 elements.  On the other hand, <C>IsomorphismSemigroups</C> may be
```

Comment:
2048 is rather optimistic...

Reply:
I've succeeded on semigroups of size 500 with 3 generators, which used about 10GB of RAM.

Reply:
Revised downwards to "a few hundred elements"